release.rb

```diff
@@ -0,0 +1,215 @@
+class ReleaseScript
+
+  def self.run(version, release_notes_file_path)
+    run_internal(version, release_notes_file_path, false)
+  end
+
+  def self.run_force(version, release_notes_file_path)
+    run_internal(version, release_notes_file_path, true)
+  end
+
+  private
+
+  def self.run_internal(version, release_notes_file_path, force)
+    remote_branch = 'convert_release_script_to_rake_task'
```

Comment:
Yeah, let's change this to `master` before merging this, and after you've tested that it all works.